### PR TITLE
feat: update request body area

### PR DESCRIPTION
**Pull request by ElinorW**
## Overview

Update request body, request header, modify permissions and access token tabs

### Demo

image
image
image

### Notes

Optional. Ancillary topics, caveats, alternative strategies that didn't work out, anything else.

## Testing Instructions

* How to test this PR
* Prefer bulleted description
* Start after checking out this branch
* Include any setup required, such as bundling scripts, restarting services, etc.
* Include test case, and expected output

### CODE REVIEW

File: src/app/views/query-runner/request/auth/AuthV9.tsx [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+    display: 'flex',
+    width: '100%',
+    minHeight: '100%',
+    justifyContent: 'center',
+    alignItems: 'center'
+  }
+});
+
+export function Auth() {
+  const { auth: { authToken }, profile, dimensions: {request:{height}}} = useAppSelector((state) => state);
```
Comment (musale) [inline src/app/views/query-runner/request/auth/AuthV9.tsx:57]: Use state values individually

File: src/app/views/query-runner/request/auth/AuthV9.tsx [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+    authenticationWrapper
+      .getToken()
+      .then((response: AuthenticationResult) => {
+        setAccessToken(response.accessToken);
+        setLoading(false);
+      })
+      .catch(() => {
+        setLoading(false);
+      });
+  }, [authToken]);
```
Comment (musale) [inline src/app/views/query-runner/request/auth/AuthV9.tsx:80]: `authToken` is never used in this `useEffect`. Remove it.

File: src/app/views/query-runner/request/headers/RequestHeadersV9.tsx [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+    minWidth: '80px'
+  },
+  listContainer: {
+    flex: 1,
+    overflow: 'auto'
+  }
+});
+
+const RequestHeaders = () => {
+  const { sampleQuery, dimensions: { request: { height } } } = useAppSelector((state) => state);
```
Comment (musale) [inline src/app/views/query-runner/request/headers/RequestHeadersV9.tsx:42]: Get state variables individually.

**Comment by ElinorW** [comment]
> This is coming along well! I tested locally and everything looks great, really good stuff. I experienced a slight issue with resizing though. In some cases, I can resize up/down and when I release, I can't resize again. In other cases, I could not trigger a resize. See this:
> 
>  resizing-request-region.mp4

Ahh this is a nice catch, I'm going to test the resizing some more

File: src/app/views/query-runner/request/auth/AuthV9.tsx (hunk 2 above)
Comment (musale) [reply src/app/views/query-runner/request/auth/AuthV9.tsx:80]: I have understood why this is like this. To trigger the useEffect to set the authToken when it's available. Leave it as is.

File: package.json [hunk 4]
```diff
@@ -5,8 +5,6 @@
         "dependencies": {
                 "@augloop/types-core": "file:packages/types-core-2.16.189.tgz",
                 "@axe-core/webdriverjs": "4.10.0",
-                "@fluentui/react-components": "9.55.1",
-                "@fluentui/react-icons": "2.0.264",
```
Comment (Mnickii) [inline package.json:9]: Oh we missed this, let's not remove the packages

**Review by Mnickii — CHANGES_REQUESTED** [review]

File: package.json (hunk 4 above)
Comment (ElinorW) [reply package.json:9]: they had been repeated

**Review by musale — APPROVED** [review]

**Review by Mnickii — APPROVED** [review]

**Pull request merged**